checkrole.yml.sample

```diff
@@ -0,0 +1,30 @@
+---
+
+# check host hardware specification against a given set of Ceph roles.
+#
+# Requires the variables to be passed in by the caller or vars defined
+# e.g.
+# vars:
+#   inventory:
+#     host1: osds,mons
+#     host2: rgws
+#     host3: mons
+#   mode: prod
+#   deployment: container
+#   osdtype: bluestore
+#   flashusage: journal 
+
+- name: Validate hosts against desired cluster state
+  hosts: all
+  become: true
+  gather_facts: false
+
+  tasks:
+  
+    - name: CEPH_CHECK_ROLE
```

Comment:
can you add this task to the ceph-validate role please? Somewhere here should be fine: https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-validate/tasks/main.yml#L5

Reply:
OK

Reply:
@leseb the module uses parms passed to it by the caller at runtime. For the ceph-installer, options selected in the UI are passed to the playbook dynamically. For ceph-validate, could I simply set the 'role' to mons,osds?

i.e.

`- name: check_role module
  ceph_check_role:
    role: 'mons,osds'`

Would that work?

Reply:
@leseb it's a bit of a misnomer. The roles in the GUI refer to the group names in ansible hence mons and osds not ceph-mon.

Reply:
Do you anything from me to get this moving? Thanks.